src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9/eventmottak/tilbakekrav/K9TilbakeKafkaStream.kt

```diff
@@ -50,14 +53,20 @@ internal class K9TilbakeKafkaStream constructor(
             builder
                 .stream(
                     fromTopic.name,
-                    Consumed.with(fromTopic.keySerde, fromTopic.valueSerde)
+                    Consumed.with(fromTopic.keySerde, Serdes.String())
                  )
-                .peek { _, e -> log.info("--> Behandlingsprosesshendelse fra k9tilbake: ${e.saksnummer} ${e.eksternId}") }
-                .foreach { _, entry ->
-                    if (entry != null) {
-                        OpentelemetrySpanUtil.span(NAME, mapOf("saksnummer" to entry.saksnummer)) {
-                            log.info("Prosesserer entry fra tilbakekreving ${entry.tryggPrint()}")
-                            TransientFeilHåndterer().utfør(NAME) { k9TilbakeEventHandler.prosesser(entry) }
+                .foreach { _, event ->
+                    if (event != null) {
+                        val tree = LosObjectMapper.instance.readTree(event)
+                        val eksternId = tree.findValue("eksternId").asText()
```

Comment:
Kan bruke tree.get i stedet for tree.findValue siden vi vet at feltene er på toppnivå. Forskjellen er at get ikke er rekursiv.